cypress/support/helper-functions/participant/participant-helpers.js

```diff
@@ -0,0 +1,22 @@
+export const timeout = Cypress.env('timeout');
+
+export function signInWithClever() {
+  cy.wait(0.2 * timeout);
+  cy.get('button', { timeout: timeout }).contains('Clever').click();
+
+  cy.origin(Cypress.env('cleverOAuthLink'), () => {
+    cy.get('input[title="School name"]', { timeout: 60000 })
+      .type(Cypress.env('cleverSchoolName'))
+      .wait(1000)
+      .type('{enter}');
```

Comment:
```suggestion
   cy.get("ul > li").contains(Cypress.env('cleverSchoolName')).click()
```
Rather than waiting, I think we can get the list that renders after searching and click the school name.